src/constants.ts

```diff
@@ -5,6 +5,8 @@ const packageDotJson = require("../package.json");
 export const PROGRAM_NAME = packageDotJson.name;
 export const PROGRAM_VERSION = packageDotJson.version;
 export const USER_AGENT = `CrossSeed/${PROGRAM_VERSION}`;
+export const TORRENT_TAG = PROGRAM_NAME;
+export const TORRENT_CATEGORY_SUFFIX = `.${PROGRAM_NAME}`;
```

Comment:
I think this is more indirection than we really need, and even if they're the same now, we could change the "stylized" version of the PROGRAM_NAME and wouldn't necessarily want to change the tag. 

```suggestion
export const TORRENT_TAG = 'cross-seed';
export const TORRENT_CATEGORY_SUFFIX = `.cross-seed`;
```